### DISCUSSION TITLE
Changing author name

### DISCUSSION THREAD

**Discussion by Berko193** [jpanther/congo#672] [Q&A]
Hello,

I cannot edit the author's name unless I edit it for all articles.
I want that any new article will get a different author name.
Editing it from the author field in a specific article doesn't change it at all.

I would appreciate your response

**Comment by jpanther** [comment]
When you say you're editing the author field on a specific article, what front matter parameter are you editing? The author is a site-wide configuration parameter and so it cannot be provided on an individual article.

**Comment by wolfspyre** [comment]
if you had a mechanism which allowed you to create an additional site-wide content-contributor, and associate certain posts with that contributor by decorating the other-author-created-page's metadata, would that solve your need?
